includes/admin.php

```diff
@@ -45,14 +44,14 @@ function enqueue_scripts( string $hook_suffix ) {
 		return;
 	}
 
-	wp_enqueue_script( 'safety-net-admin', SAFETY_NET_URL . 'assets/js/safety-net-admin.js', [ 'jquery' ], '1.0', true );
+	wp_enqueue_script( 'safety-net-admin', SAFETY_NET_URL . 'assets/js/safety-net-admin.js', array( 'jquery' ), '1.0', true );
```

Comment:
I'm not 100% sure which would be the best way to version this script... Linking it to the plugin version seems too strict (as I understand that minor changes could be applied without updating the plugin version number). 

I kinda feel the same way about generic asset versioning... 

For now, I've decided to use the `filemtime` of the file. I know that this can generate false updates when using multiple servers though...

@tommusrhodus which approach would you recommend for this specific case?